incuna-sass/helpers/_box-sizing.sass

```diff
@@ -1,8 +1,14 @@
 %border-box
-    @include box-sizing(border-box)
+    -webkit-box-sizing: border-box
+    -moz-box-sizing: border-box
+    box-sizing: border-box
```

Comment:
UPDATE: `padding-box` seems to be supported only by Firefox and yet it's not part of the specification hence I think we shouldn't be using it.